config/defaults.conf

```diff
@@ -164,7 +164,7 @@ tokenizer = "MosesTokenizer"  // The name of the tokenizer, passed to the Task c
                               // handling during data loading. It's expected that all tasks use the same
                               // tokenizer; doing otherwise will cause an error.
                               // 'MosesTokenizer' is the default. Use "" for edge probing tasks to use the
-                              // original tokenization, or one of 'OpenAI.BPE',
+                              // original tokenization, or one of 'OpenAIBPETokenizer',
```

Comment:
Can we keep `OpenAI.BPE` as supported? (we shouldn't break existing configs right before a paper deadline.)

See note on `get_tokenizer()` below - we don't need these to match class names (they won't for BERT anyway) as long as the correspondence is reasonably clear.

Reply:
Yep - I'm reverting the changes to OpenAI.BPE right now